website/ref/index.toml

```diff
@@ -1,55 +1,47 @@
 [[articles]]
 name = "language"
-title = "The Language"
+title = "The Language Specification"
 
 [[articles]]
 name = "builtin"
-title = "The Builtin Module"
+title = "<span class=module>builtin:</span> Builtin Functions and Variables"
```

Comment:
Please refrain from doing that and just use plain text title.

Reply:
Okay, but that means putting the module name at the front of the title results in a very hard to read index. So I'll revert to putting the module name at the end of the title enclosed in parentheses.